app/auth/signup/email/page.tsx

```diff
@@ -0,0 +1,113 @@
+'use client'
+import { useRef } from 'react'
+import { useForm } from 'react-hook-form'
+import {
+  Box,
+  Flex,
+  FormLabel,
+  FormErrorMessage,
+  FormControl,
+  Input,
+  Heading,
+  useToast
+} from '@chakra-ui/react'
+import { PrimaryButton } from '@/components/button'
+import { signUp } from './action'
+import { useFormResolver, SignUpSchema } from './schema'
+
+export default function SignUp() {
+  const toast = useToast()
+  const isLoading = useRef(false)
+  const toastId = useRef<ReturnType<typeof toast> | undefined>(undefined)
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors }
+  } = useForm<SignUpSchema>({
+    resolver: useFormResolver
+  })
+
+  return (
+    <>
+      <Heading>Welcome!</Heading>
+      <Heading>Create an Account</Heading>
+      <Box
+        as="form"
+        onSubmit={handleSubmit((data) => {
+          isLoading.current = true
+          signUp(data)
+            .catch((error) => {
+              if (!toastId.current) {
+                toastId.current = toast({
+                  title: "We're sorry, but you failed to sign up.",
+                  description: error.message,
+                  status: 'error',
+                  duration: 5000,
+                  isClosable: true,
+                  position: 'top',
+                  onCloseComplete() {
+                    toastId.current = undefined
+                  }
+                })
+              }
+            })
+            .finally(() => {
+              isLoading.current = false
+            })
+        })}
+      >
+        <Flex flexDirection={'column'}>
+          <FormControl isInvalid={!!errors.email} isRequired>
+            <FormLabel htmlFor="email">Email address</FormLabel>
+            <Input type="email" id="email" required {...register('email')} />
+            {errors.email && (
+              <FormErrorMessage>{errors.email.message}</FormErrorMessage>
+            )}
```

Comment:
You do not need `required` in Input, and `htmlFor` in FormLabel. 
This is HTML default required option not using ChakraUI. You also do not need id because of React hook form.
`<Input type="email"  {...register('email')} />`

ChakraUI already did it so no need htmlFor. 
`<FormLabel>Email address</FormLabel>`

The other from is same!

Reply:
Thanks , fixed !
And zod has email format checker by default, and I used it.